tests/ui/missing-function/rust-by-example-description/expected

```diff
@@ -1,2 +1,4 @@
 Status: FAILURE\
-Description: "assertion"
+Description: "Function
```

Comment:
Why not add the name of the function as well, to ensure that the demangling works corectly?

Reply:
Maybe I'm mistaken, but I was under the impression that mangled names may change across different versions of the compiler. But you're right, let's add it and we'll remove if that's the case in future toolchain updates.

Reply:
CI failed because the last (mangled) part is not consistent across different platforms. Went back to two lines but the first one includes the demangled part of the function name.

Reply:
Thanks! I think you can get rid of the hash at the end tho, see below.